transcom/mymove
# INT B-19044

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19044)

## Summary

These are to make the counts on the left nav and top nav bar more useful to the office users. Daniel did the implementation here and had some back and forth with the gov via slack, if we need to reference it.

- Counselor remarks are no longer included in the count for tasks
- Actions that block the office user from processing are now red notifications
- Other required actions are yellow

Testing AC 1:
GIVEN that I am a Service Counselor viewing "Shipments" on the "Move details" page
WHEN there are no counselor remarks,
THEN I will not be required to enter counselor remarks,
AND this item will not be included in the count of items that need my attention.
1. Create a PPM move as the customer, request an advance
2. Log in as the counselor
3. See that the counselor remarks no longer have a yellow mark and are not included in the total count
![image](https://github.com/user-attachments/assets/614435c1-0d05-49f1-b11a-b8e61cae4a64)

Testing AC 2:
GIVEN that I am a Service Counselor viewing "Shipments" on the "Move details" page
AND a customer has a PPM shipment,
WHEN the "Advance requested?" field has not been reviewed,
THEN I will see a red indicator that the item needs my attention,
AND this item is included in the count of items that need attention for the "Shipments" section in the left nav bar,
AND this item is included in the count of items that need attention for the "Move details" page in the top nav bar.
1. Use the move you just created as long as it has a requested advance (if you forgot, just make a new one)
2. Notice that since the Advance Rqeuested field has not been reviewed by the counselor, the Shipments left nav tag will have a red '1' by it.
![image](https://github.com/user-attachments/assets/081dcaee-c34a-4949-893b-6163edbb2e75)
![image](https://github.com/user-attachments/assets/3aeeabb5-59a5-4934-838e-9c295d7372c4)

Testing AC 3:
GIVEN that I am a TOO viewing "Shipments" on the "Move details" page
AND a customer has a PPM shipment that requires GHC Prime Counseling,
WHEN the "Advance requested?" field has not been reviewed,
THEN I will see a red indicator that the item needs my attention,
AND this item is included in the count of items that need attention for the "Shipments" section in the left nav bar,
AND this item is included in the count of items that need attention for the "Move details" page in the top nav bar.
1. Switch to the TOO role
2. Search for the same move you just looking at, by move code
3. Notice that since the Advance Rqeuested field has not been reviewed, and the Shipments left nav tag will have a red '1' by it.
![image](https://github.com/user-attachments/assets/d902f0a5-9236-45e8-b9d4-1d2fde26263c)

Testing AC 4:
GIVEN that I am a Services Counselor,
WHEN I am viewing "Allowances" on the "Move details" page,
THEN I will not see orange indicators for Pro-gear, Spouse pro-gear, Required medical equipment, OCIE, and Gun Safe.
1. Log in as a services counselor
2. Confirm you don't see orange indicators on the Allowances section
![image](https://github.com/user-attachments/assets/0660f409-ea54-4231-96bc-3e52842071e5)


## Screenshots
![image](https://github.com/user-attachments/assets/f0735718-b0b1-495a-a363-12efaeb10047)
![image](https://github.com/user-attachments/assets/a2638540-3423-4fe6-920d-13a09f94f23b)
